samples/ibm-samples/openscale/README.md

```diff
@@ -0,0 +1,116 @@
+# Watson OpenScale Example
+
+This simple OpenScale pipeline will demonstrate how to train a model using IBM Spark Service, then deploy it with Watson Machine Learning, and subscribe the model with Watson OpenScale monitoring.
+
+## Prerequisites
+This pipeline requires the user to have provisioned OpenScale, Spark, and Machine Learning Service on Watson, a cloud object store set up and the service credentials configured in the creds.ini file.
+
+To provision your own OpenScale, Spark, Watson Machine Learning services and cloud object store, following are the required steps.
+
+1. IBM Watson Machine Learning service instance
+
+To create a machine learning service, go to [IBM Cloud](https://cloud.ibm.com/), login with IBM account id first. From the `Catalog` page, click on `AI` tab on the left side to go to this [page](https://cloud.ibm.com/catalog?category=ai). Then click on the [`Machine Learning`](https://cloud.ibm.com/catalog/services/machine-learning) link and follow the instructions to create the service.
```

Comment:
To create a Watson Machine Learning service,

Reply:
done